src/components/ProjectDetailPage/ProjectInfo.tsx

```
@@ -36,21 +32,15 @@ export default function ProjectInfo({ projectId }: ProjectInfoProps) {
-            <div
-              className="project-detail-content mt-6 text-base text-black80 leading-relaxed"
-              dangerouslySetInnerHTML={{ __html: detail.contents.story_html }}
-            />
-          ) : (
-            <p className="mt-6 text-base text-black80 leading-relaxed">
-              프로젝트 소개글이 준비 중이에요.
-            </p>
-          )}
+          <div
+            className="project-detail-content mt-6 text-base text-black80 leading-relaxed"
+            dangerouslySetInnerHTML={{ __html: detail.storyHtml }}
```

Biome (2.3.14)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)